packages/react/src/components/navigationLink/NavigationLink.module.scss

```diff
@@ -0,0 +1,9 @@
+.navigationLink.navigationLink  {
+    --link-visited-color: none;
+    --link-color: var(--color-black-90);
+    text-decoration: none;
+}
+.navigationLink:hover {
```

Comment:
I noticed that hel.fi universal bar links have underlining also on focus:
![Screenshot 2022-11-24 at 13 10 19](https://user-images.githubusercontent.com/1610860/203771122-d3bd4905-4840-4c83-96b6-c1aacc2a7982.png)